lib/core/DocsLayout.js

```diff
@@ -23,6 +23,15 @@ class DocsLayout extends React.Component {
     if (this.props.Doc) {
       DocComponent = this.props.Doc;
     }
+    let docTitle = i18n
```

Comment:
Someone might put 

```
hide_title: false
```

in the doc header.

So, how about instead we do the converse here:

```
let docTitle = null;
// If the attribute does not exist or is set to false, we set a title.
if (!this.props.metadata.hide_title || this.props.metadata.hide_title.toLowerCase() === 'false') {
  docTitle = ....
}
```